### DISCUSSION TITLE
Where to contact me

### DISCUSSION THREAD

**Discussion by Explosion-Scratch** [Bijoujs/Bijou.js#176] [General]
Okay, so I'm Really sorry that I forgot to check discussions,  I'll try to check more often from now on, if I don't reply on GitHub for a bit don't hesitate to contact me on discord. `--Explosion--#0207`, also through my email `explosionscratch@gmail.com`

**Comment by retronbv** [comment]
(Maybe you could make a website contact form connected to a bot that dms you)

**Reply by Explosion-Scratch** [reply]
I have the feedback bot thingo
